lib/liquid/variable_lookup.rb

```diff
@@ -54,6 +54,8 @@ def evaluate(context)
         elsif @command_flags & (1 << i) != 0 && object.respond_to?(key)
           object = object.send(key).to_liquid
 
+        elsif context.strict_variables
+          raise Liquid::MissingVariable, "missing variable #{key} in #{object}"
```

Comment:
The idea is not to print exception messages into the rendered output. This message is only available in `errors` method.
And there's also a problem, that we cannot get full variable path here, only variable name and current scope. So I decided to add the scope to the message for developers to know where exactly the missing variable is.